### PR TITLE
chore: point all CKF charms to 1.8/beta channel to match release

### PR DESCRIPTION
**Pull request by DnPlas**
Point all CKF charms to their corresponding 1.8/beta channel. This PR should be merged once the charms are available in the correct track.

### CODE REVIEW

**Review by ca-scribner — CHANGES_REQUESTED** [review]
Not all charms use `channel: 1.8/*`, we need to include the track names for each charm

**Comment by ca-scribner** [comment]
We can probably use [this pr](https://github.com/canonical/bundle-kubeflow/pull/746) as a template for an easy fix here

**Comment by DnPlas** [comment]
@ca-scribner ready for a second pass, I must've just done a quick search and replace w/o noticing the versions. Please let me know what you think.

File: releases/1.8/beta/kubeflow/bundle.yaml [hunk 1]
```diff
@@ -4,53 +4,60 @@ docs: https://discourse.charmhub.io/t/3749
 applications:
   admission-webhook:
     charm: admission-webhook
-    channel: latest/beta
+    channel: 1.8/beta
     trust: true
     scale: 1
     _github_repo_name: admission-webhook-operator
+    _github_repo_branch: 1.8/beta
```
Comment (ca-scribner) [inline releases/1.8/beta/kubeflow/bundle.yaml:11]:
```suggestion
    _github_repo_branch: track/1.8
```

Comment (ca-scribner) [reply releases/1.8/beta/kubeflow/bundle.yaml:11]: I see this is the only app in the file with a `_github_repo_branch`.  Should this line either be removed, or added for all the others? iirc we use this extra field in our [scanning scripts](https://github.com/canonical/kubeflow-ci/blob/3c4f999df3de0578a035d41aae468453121c17b5/scripts/images/get-all-images.sh#L10) for our [scanning workflow](https://github.com/canonical/bundle-kubeflow/blob/413855663ed0275f42a64ecdd41c1ed602cc528d/.github/workflows/scan-images.yaml#L53)

File: releases/1.8/beta/kubeflow/bundle.yaml [hunk 2]
```diff
@@ -4,53 +4,60 @@ docs: https://discourse.charmhub.io/t/3749
...
     charm: jupyter-ui
-    channel: latest/beta
+    channel: 1.8/beta
     scale: 1
     trust: true
     _github_repo_name: notebook-operators
   katib-controller:
     charm: katib-controller
-    channel: latest/beta
+    channel: 1.8/beta
```
Comment (ca-scribner) [inline releases/1.8/beta/kubeflow/bundle.yaml:60]:
```suggestion
    channel: 0.16/beta
```

File: releases/1.8/beta/kubeflow/bundle.yaml [hunk 3]
```diff
@@ -61,19 +68,19 @@ applications:
     constraints: mem=2G
   katib-db-manager:
     charm: katib-db-manager
-    channel: latest/beta
+    channel: 1.8/beta
```
Comment (ca-scribner) [inline releases/1.8/beta/kubeflow/bundle.yaml:71]:
```suggestion
    channel: 0.16/beta
```

File: releases/1.8/beta/kubeflow/bundle.yaml [hunk 4]
```diff
@@ -145,84 +158,91 @@ applications:
     _github_repo_name: knative-operators
   kserve-controller:
     charm: kserve-controller
-    channel: latest/beta
+    channel: 1.8/beta
```
Comment (ca-scribner) [inline releases/1.8/beta/kubeflow/bundle.yaml:161]:
```suggestion
    channel: 0.11/beta
```

File: releases/1.8/beta/kubeflow/bundle.yaml [hunk 5]
```diff
@@ -145,84 +158,91 @@ applications:
...
     scale: 1
     trust: true
     _github_repo_name: oidc-gatekeeper-operator
   pvcviewer-operator:
     charm: pvcviewer-operator
-    channel: latest/beta
+    channel: 1.8/beta
     scale: 1
     trust: true
+    series: focal
```
Comment (ca-scribner) [inline releases/1.8/beta/kubeflow/bundle.yaml:215]: Is `series` required here?

**Review by ca-scribner — CHANGES_REQUESTED** [review]
mostly lgtm, but with a few comments

**Comment by DnPlas** [comment]
Thanks @ca-scribner for the review. I have updated everything, let me know what you think.

**Review by kimwnasptd — APPROVED** [review]

File: releases/1.8/beta/kubeflow/bundle.yaml (hunk 1 above)
Comment (kimwnasptd) [reply releases/1.8/beta/kubeflow/bundle.yaml]: Resolving since repo branches were added

File: releases/1.8/beta/kubeflow/bundle.yaml (hunk 5 above)
Comment (kimwnasptd) [reply releases/1.8/beta/kubeflow/bundle.yaml:245]: Resolving, as this is necessary as discussed in https://github.com/canonical/bundle-kubeflow/pull/733#discussion_r1400248494

**Pull request merged**